canton/it-lib/src/main/com/daml/CantonRunner.scala

```diff
@@ -88,6 +88,7 @@ object CantonRunner {
       s"""${participantId} {
          |      admin-api.port = ${adminPort.port}
          |      ledger-api{
+         |        max-deduplication-duration = 0s
```

Comment:
Without this there were errors about how it couldn't prune because it had a deduplication period of 168h.